src/Umbraco.Web/Models/Mapping/ContentTypeMapDefinition.cs

```diff
@@ -395,9 +399,11 @@ private static void Map(MemberPropertyTypeBasic source, MemberPropertyTypeDispla
 
             if (!(target is IMemberType))
             {
-                target.Variations = ContentVariation.Nothing;
-                if (source.AllowCultureVariant)
-                    target.Variations |= ContentVariation.Culture;
+                target.Variations = source.AllowCultureVariant
+                    // Set culture flag
+                    ? target.Variations | ContentVariation.Culture
+                    // Unset culture flag
+                    : target.Variations & ~ContentVariation.Culture;
```

Comment:
This is repeated from above, can we turn this into a small method instead so when we need to make changes, we make them the same everywhere please?

Reply:
It indeed looks identical but the involved types are not actually the same. The type of `target` is either `IContentTypeComposition` or `PropertyType` which do not seem to share any class or interface. The duplicated code just sets/unsets the `ContentVariation.Culture` based on the `AllowCultureVariant`, which originally was also duplicated of course. But I will check if it can be improved with a method.